src/Doctrine/ObjectManagerAwareTrait.php

```diff
@@ -2,27 +2,27 @@
 
 namespace Detail\Persistence\Doctrine;
 
-use Doctrine\Common\Persistence\ObjectManager as ObjectManagerInterface;
+use Doctrine\Persistence\ObjectManager;
 
 trait ObjectManagerAwareTrait
 {
     /**
-     * @var ObjectManagerInterface
+     * @var ObjectManager
      */
     protected $objectManager;
 
     /**
-     * @return ObjectManagerInterface
+     * @return ObjectManager
      */
     public function getObjectManager()
```

Comment:
Set return signature

Reply:
Don't want to introduce (other) backwards incompatible changes. We can implement type-safety when upgrading to Laminas.